src/main/java/view/InputView.java

```diff
@@ -26,6 +28,12 @@ private void validateNames(String names) {
         }
     }
 
+    private void validateUnavailableName(List<String> participantNames) {
+        if (participantNames.stream().anyMatch(name -> name.equals(InputView.EXIT))) {
```

Comment:
항상 상수를 앞쪽에 두기!
```suggestion
        if (participantNames.stream().anyMatch(name -> InputView.EXIT.equals(name))) {
```
null element가 포함되어 있다면 NPE가 날 수 있어요.
```java
    public static void main(String[] args) {
        List<String> nulls = List.of(null, null, null);
        validateUnavailableName(nulls); // NPE
    }
```